arches/app/media/js/viewmodels/resource-instance-select.js

```diff
@@ -5,19 +5,22 @@ define([
     'arches',
 ], function(ko, $, WidgetViewModel, arches) {
     var nameLookup = {};
+    require(['views/components/workflows/new-tile-step']);
```

Comment:
@ryan86 why are you requiring this module here?  require is asynchronous, so this seems dangerous.  I would just add this dependency to the list of dependencies at the top of this script.